integration_test/Dialect/Handshake/basic.mlir

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: rtl-sim
+// RUN: circt-opt %s -create-dataflow -canonicalize-dataflow -canonicalize -cse > %t0.mlir
+// RUN: circt-opt %t0.mlir -lower-handshake-to-firrtl -canonicalize -cse > %t1.mlir
+// RUN: circt-opt %t1.mlir -pass-pipeline='firrtl.circuit(firrtl.module(firrtl-lower-types))' > %t2.mlir
+// RUN: circt-translate %t2.mlir -emit-firrtl-verilog > %t3.sv
+// RUN: circt-rtl-sim.py %t3.sv %S/basic.cpp --no-default-driver
```

Comment:
And you'll need `--sim verilator` here to ensure that verilator is used. I'm working on a patch (slowly) to allow users to select the simulator they want to run for all the integration tests.